internal/pkg/runtime/engines/config/check.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package config
```

Comment:
I don't think `internal/pkg/runtime/engines/config` is the right package for a utility function which is just checking the file ownership.